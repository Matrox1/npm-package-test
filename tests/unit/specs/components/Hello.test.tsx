import * as React from "react";
import * as ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import Hello from "components/Hello";

describe("Hello", () => {
    it("renders without errors", () => {
        const component = renderer.create(
            <Hello name="Simple App" />
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});