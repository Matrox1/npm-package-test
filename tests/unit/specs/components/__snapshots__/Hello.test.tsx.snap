// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders without errors 1`] = `
<h1>
  Hello 
  Simple App
  !
</h1>
`;
